### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by ramonhpr**
**Describe what this PR introduces:** (if necessary, link currently open issues with [special keywords](https://help.github.com/en/articles/closing-issues-using-keywords))

Add github action to build and run tests. Also make the key word `PASS` and `FAILED`
 colored :smile: 
Related to #26 

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe:
Add github actions to build and tests for every commit that enter on master and for every pull request opened.

**Does this PR introduce any breaking changes?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] Related issues are referenced in the PR (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included

**Testing environment:**

- Operating System/Platform: 5.4.35-1-MANJARO
- Go version: 1.14.2

If you have relevant logs, error output and etc, please attach them.

**Other information:**

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
```
Comment (netoax) [inline .github/workflows/go.yml]: Is it ok removing this blank line to follow the same pattern of `on: push: `?

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  build:
+    name: Build and Tests
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
```
Comment (netoax) [inline .github/workflows/go.yml]: Could it be 1.14?

**Review by netoax — CHANGES_REQUESTED** [review]
Awesome, thanks for the contribution. I did just two comments to help you improve it. 🎉

File: .github/workflows/go.yml (hunk 1 above)
Comment (ramonhpr) [reply .github/workflows/go.yml]: I think it's ok. I gonna test it

File: .github/workflows/go.yml (hunk 2 above)
Comment (ramonhpr) [reply .github/workflows/go.yml]: yes

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # KNoT Babeltower
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/9140aa8c06934071ad6e3cf3b1b148ff)](https://www.codacy.com/manual/joaoaneto/knot-babeltower?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CESARBR/knot-babeltower&amp;utm_campaign=Badge_Grade)
-
+![Build](https://github.com/ramonhpr/knot-babeltower/workflows/Go/badge.svg)
```
Comment (ramonhpr) [inline README.md]: Reminder to myself: change to CESARBR beside ramonhpr

File: .github/workflows/go.yml (hunk 1 above)
Comment (ramonhpr) [reply .github/workflows/go.yml]: Done

File: .github/workflows/go.yml (hunk 2 above)
Comment (ramonhpr) [reply .github/workflows/go.yml]: Done

**Review by netoax — APPROVED** [review]
LGTM!

**Review by lcbm — APPROVED** [review]
This is awesome! Thanks @ramonhpr 😍

**Pull request merged**